pages/verify-certification-score.vue

```diff
@@ -0,0 +1,234 @@
+          </button>
+        </form>
+        <div v-if="formSent" class="form-sent">
+          <fa :icon="fas.faExclamationTriangle" />
+          <span>Il n'y a pas de certificat Pix correspondant</span>
+        </div>
+      </div>
+    </main>
+  </div>
+</template>
+
+<script>
+import { fas } from '@fortawesome/free-solid-svg-icons'
+import { documentFetcher } from '~/services/document-fetcher'
+
+export default {
+  name: 'VerifyCertificatioScore',
+  nuxtI18n: {
+    paths: {
+      'fr-fr': '/verifier-score-certification',
+      'en-gb': '/verify-certification-score',
+    },
+  },
+  async asyncData({ app, error, req, currentPagePath }) {
+    try {
+      const document = await documentFetcher(
+        app.i18n,
+        req
+      ).getVerifyCertificationForm()
+      if (process.client) window.prismic.setupEditButton()
+
+      return {
+        currentPagePath,
+        meta: document.data.meta,
+        document: document.data,
+        documentId: document.id,
+      }
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e)
+      error({ statusCode: 404, message: 'Page not found' })
+    }
+  },
+  data() {
+    return {
+      score: '',
+      code: '',
+      errors: [],
+      codeRegex: /^P-[0-9]{5}$/i,
+      scoreRegex: /^[0-9]{1,3}$/,
+      hasCodeRegexError: false,
+      hasScoreRegexError: false,
+      formSent: false,
+    }
+  },
+  computed: {
+    buttonLabel() {
+      return this.document.button_label
+    },
+    title() {
+      return this.document.title
+    },
+    verificationCodeLabel() {
+      return this.document.code_de_verification
+    },
+    candidateScoreLabel() {
+      return this.document.score_declare_du_candidat
+    },
+    getRegexErrorClass() {
+      return 'regex-error'
+    },
+    fas() {
+      return fas
+    },
+  },
+  methods: {
+    submit(e) {
+      e.preventDefault()
+      if (this.checkForm()) {
+        this.formSent = true
+      }
+    },
+    checkForm() {
+      this.hasCodeRegexError = !this.testCodeRegex()
+      this.hasScoreRegexError = !this.testScoreRegex()
+
+      return !this.hasCodeRegexError && !this.hasScoreRegexError
+    },
+    testCodeRegex() {
+      const codeIsCorrect = this.codeRegex.test(this.code)
+      return codeIsCorrect
+    },
+    testScoreRegex() {
+      const scoreIsCorrect = this.scoreRegex.test(this.score)
+      return scoreIsCorrect
+    },
+  },
+  head() {
+    const meta = this.$getMeta(this.meta, this.currentPagePath, this.$prismic)
+    return {
+      meta,
+    }
+  },
+}
+</script>
+
+<style scoped lang="scss">
+.title {
+  font-family: 'Open Sans', Arial, sans-serif;
+  font-size: 62px;
+  height: 164px;
+  line-height: 82px;
+  font-weight: 300;
+  margin-bottom: 50px;
+  max-width: 490px;
+}
+.page-body {
+  form {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    label {
+      font-family: 'Roboto', Arial, sans-serif;
+      color: rgb(23, 43, 77);
```

Comment:
```suggestion
      color: #172b4d;
```
Ou encore : 
```suggestion
      color: $blue-5;
```
+ définir $blue-5 dans `colors.scss`